safari/Config.xcconfig

```diff
@@ -1,4 +1,4 @@
-MARKETING_VERSION = 22.2.13
-CURRENT_PROJECT_VERSION = 18
+MARKETING_VERSION = 0.0.0
+CURRENT_PROJECT_VERSION = 1
```

Comment:
These are just placeholders just for Xcode and are not actually enough for the script to run. You'll have to copy them to your `LocalOverrides` file